identity/aziot-identityd/src/configext.rs

```diff
@@ -14,26 +14,46 @@ pub fn load_file(filename: &Path) -> Result<config::Settings, InternalError> {
     settings.check().map_err(InternalError::BadSettings)
 }
 
-#[derive(Debug, Eq, PartialEq, PartialOrd, serde::Deserialize, serde::Serialize)]
-pub struct HubDeviceInfo {
-    pub hub_name: String,
-
-    pub device_id: String,
-}
-
-impl HubDeviceInfo {
-    pub fn new(filename: &Path) -> Result<Option<Self>, InternalError> {
-        let info = std::fs::read_to_string(filename).map_err(InternalError::LoadDeviceInfo)?;
-
-        let info = match info.as_str() {
-            "unprovisioned" => None,
-            _ => Some(toml::from_str(&info).map_err(InternalError::ParseDeviceInfo)?),
-        };
-
-        Ok(info)
+pub fn prepare_authorized_principals(
+    principal: &[config::Principal],
+) -> (
+    std::collections::BTreeMap<config::Uid, config::Principal>,
```

Comment:
Could we clean this up a bit by importing `std::collections::{BTreeMap, BTreeSet}` at the top level? The fully qualified paths for those types are used >10 times throughout the function.

Reply:
Fixed